flexinput/src/main/java/com/lytefast/flexinput/fragment/FlexInputFragment.java

```diff
@@ -126,6 +126,9 @@ public void onItemUnselected(final Object item) {
           }
 
           private void updateUi() {
+            if (getView() == null) {
```

Comment:
by local ref you mean a FlexInputFragment field instance? This would be the same thing as that.

Or do you mean save this value so it's consistent?